test/report/test-report-uv-handles.js

```diff
@@ -2,18 +2,20 @@
 
 // Testcase to check reporting of uv handles.
 const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+const path = require('path');
 if (common.isIBMi)
   common.skip('IBMi does not support fs.watch()');
 
-if (process.argv[2] === 'child') {
-  // Exit on loss of parent process
-  const exit = () => process.exit(2);
-  process.on('disconnect', exit);
+function pipeName(windowsExtended) {
```

Comment:
This function is only called with `windowsExtended` set to `true` so the parameter is superfluous?

Reply:
Updated common.PIPE to use '\\?\' prefix. Most likely there should not be any difference regarding the pipe API. However my local windows VM doesn't behave well and lots of test cases were falling for unknown reasons. So just pushed the updates and let's see what the CI says.